_posts/2016-02-12-much-worse.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Much-Worse Jekyll Theme for academic page and blog
+date: 2016-02-12 22:00:00
+homepage: https://github.com/gchauras/much-worse-jekyll-theme
+download: https://github.com/gchauras/much-worse-jekyll-theme/archive/gh-pages.zip
+demo: http://gchauras.github.io/much-worse-jekyll-theme/
+author: Gaurav Chaurasia
+thumbnail: much-worse-jekyll-theme.jpg
+license: MIT License
+license_link: http://chibicode.mit-license.org/
```

Comment:
Thanks for the PR! This link states that the copyright holder for the theme is Shusaku Uesugi, is this correct? If not please replace it with generic http://opensource.org/licenses/MIT.